src/features/pastelPhotopea/PastelPhotopeaModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import Modal from 'react-modal'
+
+import { useAppDispatch, useAppSelector } from '../../redux/hooks'
+import PastelLoading from '../pastelLoading'
+import styles from './PastelPhotopeaModal.module.css'
+import { closePastelPhotopeaModal } from './PastelPhotopeaModalSlice'
+
+export default function PastelPhotopeaModal(): JSX.Element {
+  const { modalIsOpen } = useAppSelector(state => state.pastelPhotopeaModal)
+  const dispatch = useAppDispatch()
+  const [isLoading, setIsLoading] = useState(true)
+
+  if (!modalIsOpen) {
+    return <></>
+  }
+
+  return (
+    <Modal
```

Comment:
When the modal opens, it has a large white border. Let's remove the border to provide more space for the iframe.

Reply:
Updated